src/app/components/Button.js

```diff
@@ -1,17 +1,21 @@
-import react from 'react';
+import React from 'react';
 import { v4 as uuidv4 } from 'uuid';
+import PropTypes from 'prop-types';
 
-export default class Button extends react.Component {
-  constructor(props) {
-    super(props);
-    this.btn = this.props.btn;
-  }
+const Button = ({ click }) => {
+  const btn = ['AC', '+/-', '%', '+', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', ' ='];
```

Comment:
- [ ] Your app has 2 plus (+) signs and as a result, is missing the division sign (&#247;). Kindly fix this.

  **SUGGESTION:** Here is the HTML entity code for the sign if you wish to use it: **`\u00F7`**. To resolve this you can change this line to have something like so:
  `const btn = ['AC', '+/-', '%', '\u00F7', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', ' ='];`

  ![sahar-plus](https://user-images.githubusercontent.com/36542986/153492526-b2308f8b-bcb3-4f3d-b46d-0034af7a3d69.jpg)

Reply:
Thank you so much for your review and your valuable comments :smile:  I fixed these issues :+1: